imgui-opengl/src/jvmMain/kotlin/com/imgui/impl/ImGuiOpenGL3.kt

```diff
@@ -229,21 +268,14 @@ actual class ImGuiOpenGL3 actual constructor(
 		val lastEnableCullFace = GL30.glIsEnabled(GL30.GL_CULL_FACE)
 		val lastEnableDepthTest = GL30.glIsEnabled(GL30.GL_DEPTH_TEST)
 		val lastEnableScissorTest = GL30.glIsEnabled(GL30.GL_SCISSOR_TEST)
-		val clipOriginLowerLeft = if (useClipOrigin) {
-			val lastClipOrigin = GL30.glGetInteger(ARBClipControl.GL_CLIP_ORIGIN)
-			lastClipOrigin != GL30.GL_UPPER_LEFT
-		} else {
-			true
+		val clipOriginLowerLeft = if (!glHasClipOrigin) true else {
+			GL30.glGetInteger(ARBClipControl.GL_CLIP_ORIGIN) != GL30.GL_UPPER_LEFT
 		}
```

Comment:
This looks a bit weird.
```suggestion
		val clipOriginLowerLeft = !glHasClipOrigin || GL30.glGetInteger(ARBClipControl.GL_CLIP_ORIGIN) != GL30.GL_UPPER_LEFT
```